Makefile.uefi

```diff
@@ -16,7 +16,7 @@ EfiSmmAccess2ProtocolGuid	:= c2702b74-800c-4131-8746-8fb5b89ce4ac
 EfiSmmControl2ProtocolGuid	:= 843dc720-ab1e-42cb-9357-8a0078f3561b
 IntelRegAccessProtocolGuid	:= FD480A76-B134-4EF7-ADFE-B0E054639807
 EfiMpServiceProtocolGuid	:= 3fdda605-a76e-4f46-ad29-12f4531b3d08
-IntelPlatformTypeGuid		:= a7ced760-c71c-4e1a-acb1-89604d5216cb
+IntelPlatformTypeGuid		:= 171e9398-269c-4081-9099-3844e260466c
```

Comment:
Why?

Reply:
IntelPlatformTypeGuid is the same as EfiIioUdsProtocolGuid, so we checked the UEFI BIOS code base and found IntelPlatformTypeGuid should be 171e9398-269c-4081-9099-3844e260466c.
@morganjangwiwynn